commit 49c515179ac0d34c2bb43bf242b8eb27ff6c11b2
Author: Panu Matilainen <pmatilai@redhat.com>
Date:   Sun Oct 19 01:53:03 2008 +0300

    Permit tab as whitespace between macro arguments (rhbz#467567)
    - another dumb regression from macro handling cleanups

diff --git a/rpmio/macro.c b/rpmio/macro.c
index 1e292c6..d1cbf93 100644
--- a/rpmio/macro.c
+++ b/rpmio/macro.c
@@ -798,7 +798,7 @@ grabArgs(MacroBuf mb, const rpmMacroEntry me, const char * se,
 	char *s = xcalloc((lastc-se)+1, sizeof(*s));
 	memcpy(s, se, (lastc-se));
 
-	argvSplit(&av, s, " ");
+	argvSplit(&av, s, " \t");
 	argvAppend(&argv, av);
 
 	argvFree(av);
